doc/flatpak-build-update-repo.xml

```diff
@@ -164,6 +164,16 @@
                 </para></listitem>
             </varlistentry>
 
+            <varlistentry>
+                <term><option>--deploy-sideload-collection-id</option></term>
```

Comment:
Do we really need this? Collection IDs have been enabled client side on Endless computers for years without issues.

Reply:
I would like to enable it by default on flathub for 1.7 clients, but not for the pre 1.7 clients where we so far have not enabled it. Mainly because why enable it and deal with whatever (if any) fallout for something that we don't want to maintain longterm. 

I realize you already deploy collection ids and will need to fix any outstanding bugs anyway, but I still think its unnecessary to enable it globally "by accident" if we want to enable the sideload stuff only.